cmd/internal/ec2/ec2.go

```diff
@@ -49,6 +52,8 @@ func init() {
 	// initialize common apidump flags as flags for the ecs add command
 	apidumpFlags = apidump.AddCommonApiDumpFlags(Cmd)
 
+	SetupInEC2Cmd.PersistentFlags().BoolVarP(&forceOverwrite, "force", "f", false, "If the service files already exists, overwrite them and don't prompt the user")
```

Comment:
```suggestion
	SetupInEC2Cmd.PersistentFlags().BoolVarP(&forceOverwrite, "force", "f", false, "If the service files already exist, overwrite them and don't prompt the user")
```

The 'user' here is presumably the one running the command -- so maybe this could be rewritten to focus on what it _does_ rather than what it does _not_ do.

"Overwrite existing service files without asking for confirmation"
"Skip confirmation check before overwriting existing service files"

might be a more compact way of writing it.